ocp_resources/datavolume.py

```diff
@@ -99,42 +92,27 @@ def __init__(
         DataVolume object
 
         Args:
-            name (str): DataVolume name.
-            namespace (str): DataVolume namespace.
             source (str): source of DV - upload/http/pvc/registry.
             size (str): DataVolume size - format size+size unit, for example: "5Gi".
             storage_class (str, default: None): storage class name for DataVolume.
             url (str, default: None): url for importing DV, when source is http/registry.
-            content_type (str, default: "kubevirt"): DataVolume content type.
-            access_modes (str, default: None): DataVolume access mode.
-            cert_configmap (str, default: None): name of config map for TLS certificates.
-            secret (Secret, default: None): to be set as secretRef.
-            client (DynamicClient): DynamicClient to use.
-            volume_mode (str, default: None): DataVolume volume mode.
-            hostpath_node (str, default: None): Node name to provision the DV on.
-            source_pvc (str, default: None): PVC name for when cloning the DV.
-            source_namespace (str, default: None): PVC namespace for when cloning the DV.
-            multus_annotation (str, default: None): network nad name.
-            bind_immediate_annotation (bool, default: None): when WaitForFirstConsumer is set in  StorageClass and DV
-            should be bound immediately.
-            preallocation (bool, default: None): preallocate disk space.
-            teardown (bool, default: True): Indicates if this resource would need to be deleted.
-            privileged_client (DynamicClient, default: None): Instance of Dynamic client
-            yaml_file (yaml, default: None): yaml file for the resource.
-            delete_timeout (int, default: 4 minutes): timeout associated with delete action.
-            api_name (str, default: "pvc"): api used for DV, pvc/storage
-            delete_after_completion (str, default: None): annotation for garbage collector - "true"/"false"
+            content_type (str, optional): DataVolume content type. Default value is "kubevirt".
+            access_modes (str, optional): DataVolume access mode. Default value is None.
```

Comment:
```suggestion
            access_modes (str, optional): DataVolume access mode. 
```
Either keep `(str, default: None)` convention or remove